IPs/RSKIP351.md

```diff
@@ -0,0 +1,68 @@
+---
+rskip: 351
+title: Miniheader: block header compression
+status: Draft
+purpose: Sca
+author: IO (ilan@iovlabs.org)
+layer: Core
+complexity: 1
+requires: 144, 194
+created: 2022-09-12
+---
+
+## Abstract
+
+This RSKIP proposes a technique to compress the block header. By moving information such as the logs bloom to a new data structure, the block header can be modified and versioned including new information without changing the block header format.
```

Comment:
I understood that the versioning is for the extended block header

Reply:
No, we use a version in the header to know how to read the extension. No?